src/frontend/ui/ui_manager.rs

```diff
@@ -190,17 +202,23 @@ impl UiManager {
     fn save_to_clipboard(&self) {
         let img = self.get_crop_image();
 
-        let mut child = if crate::backend::is_wayland() {
-            std::process::Command::new("wl-copy")
-                .stdin(Stdio::piped())
-                .spawn()
-                .expect("Couldn't spawn wl-copy process")
-        } else {
-            std::process::Command::new("xclip")
-                .args(["-selection", "clipboard", "-target", "image/png"])
+        let mut child = {
+            let (clip_man, args) = if crate::backend::is_wayland() {
+                (&self.config.wayland.clip_man, &self.config.wayland.args)
+            } else {
+                (&self.config.x11.clip_man, &self.config.x11.args)
+            };
+
+            std::process::Command::new(clip_man)
+                .args(args)
                 .stdin(Stdio::piped())
                 .spawn()
-                .expect("Couldn't spawn xclip process")
+                .unwrap_or_else(|err| {
+                    panic!(
+                        "Coulnd't spawn '{}'. Is it a typo or did you really install it?\n{}",
+                        &clip_man, err
+                    )
+                })
```

Comment:
I really don't feel comfortable with that as a malicious config file can potentially harm the entire system by injecting malicious code.

Reply:
That's a good point...
Should I ask on a forum for other opinions?

Reply:
https://softwareengineering.stackexchange.com/questions/450994/allow-command-execution-in-config-file/450995#450995